### PR TITLE
Add chart for policies shared for all installations

### PR DESCRIPTION
**Pull request by MarcelMue**

### CODE REVIEW

File: hack/setup-local.sh [hunk 1]
```diff
@@ -14,4 +14,4 @@ kubectl create --context kind-kyverno-cluster -f https://raw.githubusercontent.c
 # CAPZ CRDs
 kubectl create --context kind-kyverno-cluster -f https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/master/config/crd/bases/infrastructure.cluster.x-k8s.io_azureclusters.yaml
 # Kyverno
-kubectl create --context kind-kyverno-cluster -f https://raw.githubusercontent.com/kyverno/kyverno/main/definitions/release/install.yaml
+kubectl create --context kind-kyverno-cluster -f https://raw.githubusercontent.com/kyverno/kyverno/v1.4.1/definitions/release/install.yaml
```
Comment (MarcelMue) [inline hack/setup-local.sh:17]: This fixes the test cases. Seems like an upstream regression.

**Review by QuentinBisson — APPROVED** [review]
LGTM

**Review by stone-z — COMMENTED** [review]
LGTM but you'll have to also add it to the `config` repo

File: .circleci/config.yml [hunk 2]
```diff
@@ -108,3 +113,15 @@ workflows:
           on_tag: false
           requires:
             - test-policies
+      # Shared
+      - architect/push-to-app-catalog:
+          name: push-policies-shared-to-catalog
+          app_catalog: control-plane-test-catalog
```
Comment (QuentinBisson) [inline .circleci/config.yml:119]: @MarcelMue is there a reason why we only deployed it on the test catalog ?

Comment (MarcelMue) [reply .circleci/config.yml:119]: No - I need to clean this up

Will fix `config` repo and this push throughout the day

Comment (QuentinBisson) [reply .circleci/config.yml:119]: No worries :D 

Would it be enough https://github.com/giantswarm/config/pull/365  ?

**Pull request merged**